### PR TITLE
MHRProcess periodNo is set to context with name period

### PR DESCRIPTION
**Pull request by SusanneCalderon**
Fixes https://github.com/adempiere/adempiere/issues/3004

### CODE REVIEW

**Comment by e-Evolution** [comment]
@SusanneCalderon , we have here a problem, because from that Payroll functionality was created, we always used _Period, so if you change name, a lot the  payrolls can throw errors calculated payroll because  uses this syntaxis, so I think no is good idea change or rename, but if you  are your preference can create _PeriodNo as an alias too

**Comment by SusanneCalderon** [comment]
@e-Evolution 
Ok, will do it.

**Comment by SusanneCalderon** [comment]
Hi @e-Evolution .
So I switched back to the original code and for further use I created two new alias, _PeriodNo and _PeriodID.

File: org.eevolution.hr_and_payroll/src/main/java/base/org/eevolution/model/MHRProcess.java [hunk 1]
```diff
@@ -977,6 +977,8 @@ private void createMovements() throws Exception
 		scriptCtx.put("_From", dateFrom);
 		scriptCtx.put("_To", dateTo);
 		scriptCtx.put("_Period", payrollPeriod.getPeriodNo());
+		scriptCtx.put("_PeriodNo", payrollPeriod.getPeriodNo());
```
Comment (yamelsenih) [inline org.eevolution.hr_and_payroll/src/main/java/base/org/eevolution/model/MHRProcess.java]: Hi @SusanneCalderon I suggest change it according with #761
The **_Period** is representative of **PeriodNo** and for old compatibility is better keep, but you can add the follow keys I think:

Example:
**_HR_Period_ID**: Period Id

The problem was that in beginning the keys was created without a good standard definition but I think that we should keep old implementation wihtout problems

Comment (SusanneCalderon) [reply org.eevolution.hr_and_payroll/src/main/java/base/org/eevolution/model/MHRProcess.java:980]: Hi @yamelsenih 
Theese are the keys now for period values:

		scriptCtx.put("_Period", payrollPeriod.getPeriodNo());
		scriptCtx.put("_PeriodNo", payrollPeriod.getPeriodNo());
		scriptCtx.put("_PeriodID", getHR_Period_ID());

**Review by yamelsenih — CHANGES_REQUESTED** [review]
@SusanneCalderon please change it for merge

**Comment by e-Evolution** [comment]
@SusanneCalderon  please only change the variable to _HR_Period_ID vs _PeriodID because is more clear that we use the payroll period id and not other period

**Pull request merged**